server/common/deliverable/get.ts

```diff
@@ -4,15 +4,14 @@ import { Sequelize } from "sequelize";
 import models from "db/models";
 
 import paginate from "helpers/paginate";
-import { resJsonMessage } from "helpers/res-json-message";
 
 interface propsWhere {
   userId?: string | string[];
   issueId?: string | number;
 }
 
 export default async function get(req: NextApiRequest, res: NextApiResponse) {
-  const { address, issueId } = req.query;
+  const { address, issueId, page, sortBy, order } = req.query;
   const where = {} as propsWhere;
 
   if (address) {
```

Comment:
we should have a relation between users and pull-requests and you should simply include the relation on the query on line 43